src/hashbang.sh

```diff
@@ -167,9 +167,18 @@ done
 
 if [ "x$key" = "x" ]; then
     if ask " Do you want us to generate a key for you?" Y; then
-        ssh-keygen -t rsa -C "#! $username"
-        keyfile="~/.ssh/id_rsa.pub"
-        key=$(cat ~/.ssh/id_rsa.pub)
+        echo -n "Path to file (~/.ssh/id_rsa): ";
```

Comment:
Two thoughts here.
1. What if they specify a totally invalid file here? We won't catch it until ssh-keygen blows up, then we are hosed
2. Most people don't even know what a file path or ssh key is at this point, better to provide a default then give advanced users an option to override it. e.g "Create key at default location? (~/.ssh/id_rsa)" then if yes, use it, if no, then ask for custom path.